diesel/src/pg/types/mod.rs

```diff
@@ -432,9 +454,13 @@ pub mod sql_types {
     ///
     /// - [`ipnetwork::IpNetwork`][IpNetwork]
     ///
-    /// [`ToSql`]: ../../../serialize/trait.ToSql.html
-    /// [`FromSql`]: ../../../deserialize/trait.FromSql.html
-    /// [IpNetwork]: ../../../../ipnetwork/enum.IpNetwork.html
+    /// [`ToSql`]: crate::serialize::ToSql
+    /// [`FromSql`]: crate::deserialize::FromSql
+    #[cfg_attr(feature = "ipnetwork", doc = " [IpNetwork]: ipnetwork::IpNetwork")]
+    #[cfg_attr(
+        not(feature = "ipnetwork"),
+        doc = " [IpNetwork]: ../../../../ipnetwork/enum.IpNetwork.html"
```

Comment:
Isn't that link broken if the feature is not enabled?

Reply:
Thank you. I missed it.

Reply:
```suggestion
        doc = " [IpNetwork]: https://docs.rs/ipnetwork/0.17.0/ipnetwork/enum.IpNetwork.html"
```